velox/exec/fuzzer/JoinFuzzer.cpp

```diff
@@ -249,7 +254,8 @@ class JoinFuzzer {
       const std::vector<RowVectorPtr>& probeInput,
       const std::vector<RowVectorPtr>& buildInput,
       const std::vector<std::string>& outputColumns,
-      std::vector<PlanWithSplits>& altPlans);
+      std::vector<PlanWithSplits>& altPlans,
+      const std::string& filter = "");
```

Comment:
nit: The default argument value is not needed?

Reply:
Done.